### ISSUE TITLE
phsc.cmd.phyloscanner.multi

### ISSUE THREAD

**Issue by antoine4ucsd** [olli0601/Phyloscanner.R.utilities#3]
Hello
I tried to apply your nice tutorial to my data. 
I started creating the bash scripts using the same parameters with 6 bam files and their own reference.
everything goes well for crating the pty.runs but I have the following error when running
```
pty.c <- phsc.cmd.phyloscanner.multi(pty.runs, pty.args)

The following objects are masked from pty.args (pos = 11):

    alignments.file, alignments.pairwise.to, alignments.root, all.bootstrap.trees, data.dir, dont.check.duplicates, dont.check.recombination, keep.overhangs, mem.save,
    merge.paired.reads, merge.threshold, min.internal.quality, min.read.count, no.trees, num.bootstraps, out.dir, prog.mafft, prog.pty, quality.trim.ends, select, verbose,
    win, window.automatic, work.dir

Error in phsc.cmd.phyloscanner.one(pty.args, file.input, file.patient) : 
  object 'default.coord' not found
```

anything I am missing?

**Comment by olli0601** [comment]
Hi,

The "default.coord" sets nucleotide positions in HXB2 that should be excised from the alignments, typically because they are sites associated with major drug resistance mutations, and keeping them might confound phylogeny reconstruction.

@abriz97 I am not sure if you or XX added this feature - the excision positions are hard coded, and that does not seem the best solution?

Antoine, it is OK to set this to NA for a first pass.

olli

**Comment by abriz97** [comment]
Hi both,

The "default.coord" was present in the first version of `utility.R` pushed by Xiaoyue.
I do agree that the parametrisation is awkard as setting `default.coord=FALSE` doesn't allow you to specify the excision cohordinates.
I assumed Xiaoyue's rationale was that users may only be interested in excising a couple possible set of cohordinates.

If this is not the case, I could either change the type of  `default.coord` to an integer vector (directly specifying the cohordinates), or maybe achieve the same with an additional arguments.

Is this the desired behaviour?

**Comment by olli0601** [comment]
I think best to set a default value for "default.coord" in the R function header, and users can then override that with NA?

**Comment by antoine4ucsd** [comment]
not sure if it helps but it is working by adding this in the code upfront
default.coord=NA
discard.improper.pairs=NA
realignment=T

**Comment by abriz97** [comment]
Hi all, I now pushed some minor changes on the `utility.R` script which should take care of this. 

I added the 3 lines mentioned by @antoine4ucsd at the start of `phsc.cmd.phyloscanner.one` function definition, which can then be overwritten by the user through the `attach(pty.args)` line.
I think the intended behaviour by Xiaoyue was to specify these in the parameters in the `pty.args` argument list, as for example we do in `make_deep_sequence_alignments.R` script.
